test/Sentry.AspNetCore.Tests/Extensions/HttpContextExtensionsTests.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Sentry.AspNetCore.Extensions;
+using Xunit;
+
+namespace Sentry.AspNetCore.Tests.Extensions
+{
+    public class HttpContextExtensionsTests
+    {
+#if !NETCOREAPP2_1
```

Comment:
Can this be defined the whole file instead?

Reply:
sure